.ci/end2end.groovy

```diff
@@ -123,3 +142,7 @@ pipeline {
     }
   }
 }
+
+def notifyStatus(String description, String status) {
+  withGithubNotify.notify('end2end-for-apm-ui', description, status, getBlueoceanDisplayURL())
```

Comment:
This method has been created to help to use a groovy variable with another method different than `call`, otherwise I'd be required to use the `scripts` as the declarative pipeline doesn't allow to use `groovy-var.method` :(